ui/src/main/js/distribution/Index.js

```diff
@@ -336,17 +364,62 @@ class Index extends Component {
         return false;
     }
 
+    onPageChange(page, sizePerPage) {
+        this.setState({
+            currentPage: page,
+            sizePerPage: sizePerPage
+        });
+    }
+
+    onSizePerPageListChange(sizePerPage) {
+        this.setState({
+            currentPageSize: sizePerPage
+        });
+    }
+
+    onSearchChange(searchText, colInfos, multiColumnSearch) {
+        // Not currently supported
+    }
+
+    onSortChange(sortName, sortOrder) {
+        // Not currently supported
+    }
+
+    onRowClick(row) {
+        // Do nothing for single-click (this will allow rows to be "checked" on single-click for bulk operations)
+    }
```

Comment:
Are these TODOs? If so, we should add that. If they are meant to be empty for now, you should be able to put an empty function directly into wherever they are used.

Reply:
They are being addressed by a separate ticket, so I wanted to leave them for ease of implementation.